projects/scully-disable-angular/src/lib/index.ts

```diff
@@ -4,10 +4,10 @@ import { existsSync, readFileSync, writeFileSync } from 'fs';
 import { join } from 'path';
 
 const disableAngularPlugin = async (html) => {
-  const statsJsonPath = join(scullyConfig.distFolder, 'stats.json');
+  const statsJsonPath = join(scullyConfig.distFolder, 'stats-es2015.json');
```

Comment:
shouldn't we check for `stats.json` AND `stats-es2015.json`?

Reply:
Is there a scenario that stats.json is generated? I can't reproduce here.

Reply:
changing the `target` in `tsconfig.json` from `es2015` to `es5` changes the output name of stats.
`stats-es2015.json` becomes `stats.json`